source/user-manual/agents/query-configuration.rst

```diff
@@ -7,29 +7,29 @@ Query configuration
 
 The actual configuration of an agent, or the manager can be queried on demand by clicking on the *Agents* tab or the *Management* tab. From here you will be able to fetch the active configuration in real time.
 
-.. thumbnail:: ../../../images/kibana-app/features/query-configuration/configuration-section.png
```

Comment:
- Use the Wazuh dashboard, instead of Kibana, for the screenshots. 

- Consider using a shorter, more generic, agent name. 

- Use bold formatting, instead of italic, for the WUI elements. For example, click on **Management**.  
See the Wazuh Style Guide for Technical Writing.


![image](https://user-images.githubusercontent.com/61882981/164478887-1b16a3e6-7cb8-4e72-bd87-b2f4d1f81e51.png)